From 5df84f6d7b86475a371b3da4b4ac603cb86964a6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Thu, 24 Apr 2025 19:40:12 +0000
Subject: [PATCH] window: Don't set single-click-activate on columnview

It causes that the selected row is changing on hover while the popover
menu is open, causing that the selected action is applied on wrong row.

Closes: https://gitlab.gnome.org/GNOME/baobab/-/issues/157
---
 data/ui/baobab-main-window.ui | 1 -
 1 file changed, 1 deletion(-)

diff --git a/data/ui/baobab-main-window.ui b/data/ui/baobab-main-window.ui
index 379135db..d7f1ac70 100644
--- a/data/ui/baobab-main-window.ui
+++ b/data/ui/baobab-main-window.ui
@@ -121,7 +121,6 @@
                                           <object class="GtkColumnView" id="columnview">
                                             <property name="hexpand">True</property>
                                             <property name="reorderable">False</property>
-                                            <property name="single-click-activate">True</property>
                                             <signal name="activate" handler="columnview_activate"/>
                                             <property name="model">
                                               <object class="GtkSingleSelection" id="columnview_selection">
-- 
GitLab

